docs/staff/editor/universe.md

```diff
@@ -0,0 +1,166 @@
+
+![Not the owner of the universe message](/img/staff/editor/universe/not-the-owner-message.png)
+
+## Editing the universe
+
+If you are the owner of your universe, you can edit it. You can edit the universe by switching the edit mode switch to on.
+
+![Edit mode switch](/img/staff/editor/universe/edit-mode-switch.png)
+
+After this you can click and drag on modules or other cells to move them around. To save the current layout, you can press the save button in the top left.
+
+You can also resize almost all cells, you do this by first clicking on the cell, selecting it, and dragging the small gray circle on the bottom right.
+
+To remove a cell, click on the cell, selecting it, and clicking the red `X`.
+
+![Moving a module](/img/staff/editor/universe/move-module.gif)
+
+### Adding cells to the universe
+
+#### Modules
+
+To add a module to the universe, open the menu on the bottom right by clicking on the red circle menu.
+Click on `Connect module`, this will open a dialog where you can search for a module to connect to your universe.
+
+#### Level gates
+
+To add a level gate to the universe, open the menu on the bottom right by clicking on the red circle menu.
+Click on `New level gate`, this will open a dialog where you can set the level for the level gate.
+
+#### Groups
+
+Groups can be used to visually group certain modules together. We have 2 types of groups, a rectangle, and a circle.
+These can be added to the universe by clicking on them.
+
+![Tools groups](/img/staff/editor/universe/tools-groups.png)
+
+Groups also move a bit differently, because they can stack on top of each other.
+To move a group, you first have to select it, then unlock it by clicking on the lock in the top right, after this you can move the group around.
+
+![Moving a group](/img/staff/editor/universe/move-group.gif)
+
+#### Labels
+
+Labels can be used to add text to the universe, these are often used together with a [group](#groups). You can add a label by clicking on the `Tt` icon.
+
+![Tools label](/img/staff/editor/universe/tools-label.png)
+
+### Lines and flow
+
+In order to create a flow inside your universe that the students will follow, you need to connect modules together using lines.
+This will make the selected module a prerequisite for the module it is connected to. This means that the current module has to be completed before the student can continue.
+
+For example, here the student first has to complete the module `HTML / CSS - Beginner` before the student can start on `HTML / CSS - Advanced`
+
+![Connect module](/img/staff/editor/universe/connect-module.gif)
+
+To remove a line between 2 modules, you need to hover over the line, and press the red `X` button.
+
+### Start module
+
+The start module can be selected using the dropdown menu in the top right. This is only used for the student's tour. Not the start of the universe.
```

Comment:
Dat is niet helemaal waar, je wordt voor die module ingeschreven tijdens de tour.

Dus als student wordt je behoorlijk aangemoedigd daar te beginnen

Reply:
Dit stukje heb ik een beetje herschreven